### PR TITLE
issue-15-update-docs

### PR DESCRIPTION
**Pull request by CarlaLlama**

### CODE REVIEW

**Comment by hodgestar** [comment]
We don't need the `README.html` file -- that must have been accidentally generated when were trying to figure out what was breaking in `README.rst`. We should delete `README.html` (using `git rm`) either in this PR or a later one.

File: vumi_http_proxy/config_reader.py [hunk 1]
```diff
@@ -1,8 +1,19 @@
+"""Reader class for specified .yml configuration file
+    proxy-blacklist: [list of ip addresses to blacklist]
```
Comment (hodgestar) [inline vumi_http_proxy/config_reader.py]: You can do:

```
:type proxy-blacklist: list of strings
:param proxy-blacklist: List of ip addresses to blacklist
```

to get a nicely formatted parameter list.

Comment (hodgestar) [inline vumi_http_proxy/config_reader.py]: I see you already used this lower down! Woot. :)

**Comment by hodgestar** [comment]
Left a couple of minor comments, but looks good!

**Comment by CarlaLlama** [comment]
Thanks! :hamster:

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**